app/controllers/common-service.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const isOnline = require('is-online');
```

Comment:
This file seems kinda silly when we can just import the `is-online` module wherever we need it.